### PR TITLE
Fix value editor icons

### LINKED ISSUE #385: Pyface icons not found by the qt TreeEditor
The qt `TreeEditor` doesn't find the zipped [icons](https://github.com/enthought/pyface/blob/master/pyface/image/library/icons.zip) from pyface. Resulting in a `ValueEditor` which doesn't work properly, because it depends on those icons.

### PR DESCRIPTION
**Pull request by corranwebster**
This fixes #385.  Includes some drive-by clean-up.

### CODE REVIEW

**Comment by corranwebster** [comment]
@martinRenou this should fix the issue you saw with the ValueEditor.

File: traitsui/qt4/tree_editor.py [hunk 1]
```diff
@@ -23,7 +23,8 @@
 
 from pyface.qt import QtCore, QtGui
 
-from pyface.resource_manager import resource_manager
+from pyface.api import ImageResource
+from pyface.ui_traits import convert_image
```
Comment (martinRenou) [inline traitsui/qt4/tree_editor.py]: I have an `ImportError: No module named ui_traits` here, with `pyface==5.1.0`, which is the latest version apparently.

Comment (corranwebster) [reply traitsui/qt4/tree_editor.py]: You'll need Pyface master for current TraitsUI code to work.

Comment (martinRenou) [reply traitsui/qt4/tree_editor.py]: Ok thanks

**Pull request merged**